File: data/src/main/java/com/thinkzi/weather/data/model/Rain.java
package com.thinkzi.weather.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rain {

    @SerializedName("rainVolume")
    @Expose
    private double rainVolume;

    public Rain() {
        this.rainVolume = 0.0;
    }

    public double getRainVolume() {
        return rainVolume;
    }

    public void setRainVolume(double _rainVolume) {
        this.rainVolume = rainVolume;
    }

}

File: data/src/main/java/com/thinkzi/weather/data/model/Rain.kt
package com.thinkzi.weather.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Rain {

    @SerializedName("rainVolume")
    @Expose
    internal var rainVolume: Double = 0.toDouble()

    init {
        this.rainVolume = 0.0
    }

    fun getRainVolume(): Double {
        return rainVolume
    }

    fun setRainVolume(_rainVolume: Double) {
        this.rainVolume = _rainVolume
    }

}
